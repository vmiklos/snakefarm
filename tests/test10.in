debug 1
out 1
time 100
load test10.map
addplayer 1
addsnake 1 1 3 1 1 0
12 0
13 0
20 0
19 0
18 0
endsnake
step
exit