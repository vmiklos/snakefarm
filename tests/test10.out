Event MapLoad test10.map
Event Step 1
StepEvent Snake_Snake 1
StepEvent SnakeDie
End Step
Event Exit